app/javascript/react/components/MonsterShow.js

```diff
@@ -0,0 +1,60 @@
+import { Binding } from "@babel/traverse";
+import React, { useState, useEffect } from "react";
+
+const MonsterShow = (props) => {
+
+  const [monster, setMonster] = useState({})
+
+  const getMonster = async () => {
+    try{
+      let monsterId = props.match.params.id
+      const response = await fetch(`/api/v1/monsters/${monsterId}`)
```

Comment:
Is this currently working?  It doesnt look like there is a route for this path.

Reply:
We fixed that now.